examples/dynamo/eg3/solver_mod.x90

```diff
@@ -437,7 +437,7 @@ contains
   call invoke_set_field_scalar(1.0_r_def, Ax)
   call invoke( matrix_vector_kernel_mm_type(lumped_weight,Ax,mm) )
 
-  !PSY call invoke ( divide_field(rhs, lumped_weight, lhs))
+  !PSY call invoke ( divide_fields(rhs, lumped_weight, lhs))
```

Comment:
Thanks, could you uncomment any others in the example as well and as merge them into single invoke calls when they are next to each other. Thanks!